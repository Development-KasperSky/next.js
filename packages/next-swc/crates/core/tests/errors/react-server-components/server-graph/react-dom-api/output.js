import { findDOMNode, flushSync, unstable_batchedUpdates } from "react-dom";
export default function() {
    return null;
}
