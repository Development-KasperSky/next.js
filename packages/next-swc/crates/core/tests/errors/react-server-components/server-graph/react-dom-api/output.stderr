
  x Disallowed ReactDOM API `findDOMNode` in the Server Components compilation.
   ,-[input.js:2:5]
 2 | findDOMNode,
   : ^^^^^^^^^^^
   `----

  x Disallowed ReactDOM API `flushSync` in the Server Components compilation.
   ,-[input.js:3:3]
 3 | flushSync,
   : ^^^^^^^^^
   `----

  x Disallowed ReactDOM API `unstable_batchedUpdates` in the Server Components compilation.
   ,-[input.js:4:3]
 4 | unstable_batchedUpdates,
   : ^^^^^^^^^^^^^^^^^^^^^^^
   `----
